routes.js

```diff
@@ -33,7 +33,7 @@ function fastifySwagger (fastify, opts, next) {
 
   // serve swagger-ui with the help of fastify-static
   fastify.register(require('fastify-static'), {
-    root: resolve('./static'),
+    root: resolve(__dirname, 'static'),
```

Comment:
you might want to use `path.join(__dirname, 'static')` instead. I'm good with no test in that case.

Reply:
done, thank you for the suggestion